core/flyout_base.js

```diff
@@ -1251,4 +1249,29 @@ Flyout.prototype.getX;
  */
 Flyout.prototype.getY;
 
+/**
+ * Scroll the flyout to the beginning of its contents.
+ */
+Flyout.prototype.scrollToStart;
+
+/**
+ * The type of a flyout content item.
+ * @enum {string}
+ */
+Flyout.FlyoutItemType = {
+  BLOCK: 'block',
+  BUTTON: 'button',
+};
+
+/**
+ * A flyout content item.
+ * @typedef {{
+ *    type: !Flyout.FlyoutItemType,
+ *    button: (!FlyoutButton|undefined),
+ *    block: (!BlockSvg|undefined),
+ * }}
+ */
+Flyout.FlyoutItem;
```

Comment:
Export and use wherever else you need it, instead of adding it to the class. That way it's not re-exported out to the world.

Reply:
Done.